tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py{27,36,37,38,39}
+skipsdist = True
+skip_missing_interpreters = True
+
+[testenv]
+deps =
+    numpy
+setenv =
+    SDL_VIDEODRIVER=dummy
+    SDL_AUDIODRIVER=disk
+passenv =
+    PORTMIDI_INC_PORTTIME
+commands =
+    python -m buildconfig -auto
```

Comment:
I wonder if this `python -m buildconfig -auto` line is necessary?

Reply:
It's needed if system is missing some dependencies. I presumed we want to let people also test how things work when some of the libs are missing.